package.json

```diff
@@ -1,62 +1,62 @@
+    "test": "ng test",
+    "skyux:generate-lib-resources-module": "ng generate @skyux/i18n:lib-resources-module",
+    "lint": "ng lint",
+    "skyux:generate-documentation": "ng generate @skyux-sdk/documentation-schematics:documentation",
+    "postbuild": "npm run skyux:generate-documentation"
   },
-  "keywords": [],
-  "author": "Blackbaud",
-  "license": "ISC",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/blackbaud/skyux-router.git"
-  },
-  "bugs": {
-    "url": "https://github.com/blackbaud/skyux-router/issues"
-  },
-  "homepage": "https://github.com/blackbaud/skyux-router#readme",
-  "peerDependencies": {
-    "@angular/common": "^9.1.6",
-    "@angular/core": "^9.1.6",
-    "@angular/router": "^9.1.6",
-    "@skyux/config": "^4.3.0",
-    "@skyux/core": "^4.0.0"
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "12.2.7",
+    "@angular/common": "12.2.7",
+    "@angular/compiler": "12.2.7",
+    "@angular/core": "12.2.7",
+    "@angular/forms": "12.2.7",
+    "@angular/platform-browser": "12.2.7",
+    "@angular/platform-browser-dynamic": "12.2.7",
+    "@angular/router": "12.2.7",
+    "@skyux/assets": "5.0.0",
+    "@skyux/config": "5.0.0-beta.4",
+    "@skyux/core": "5.0.0-beta.11",
+    "@skyux/i18n": "5.0.0",
+    "@skyux/indicators": "5.0.0-beta.4",
+    "@skyux/modals": "5.0.0-beta.6",
+    "@skyux/theme": "5.0.0",
+    "rxjs": "6.6.7",
+    "tslib": "2.3.1",
+    "zone.js": "0.11.4"
   },
-  "dependencies": {},
   "devDependencies": {
-    "@angular/animations": "9.1.13",
-    "@angular/common": "9.1.13",
-    "@angular/compiler": "9.1.13",
-    "@angular/compiler-cli": "9.1.13",
-    "@angular/core": "9.1.13",
-    "@angular/forms": "9.1.13",
-    "@angular/platform-browser": "9.1.13",
-    "@angular/platform-browser-dynamic": "9.1.13",
-    "@angular/router": "9.1.13",
-    "@blackbaud/auth-client": "2.50.0",
-    "@skyux-sdk/builder": "4.12.0",
-    "@skyux-sdk/builder-plugin-skyux": "4.2.0",
-    "@skyux-sdk/e2e": "4.1.0",
-    "@skyux-sdk/testing": "4.3.0",
-    "@skyux/assets": "4.0.1",
-    "@skyux/config": "4.4.0",
-    "@skyux/core": "4.8.0",
-    "@skyux/http": "4.2.0",
-    "@skyux/i18n": "4.4.0",
-    "@skyux/indicators": "4.11.0",
-    "@skyux/modals": "4.6.0",
-    "@skyux/omnibar-interop": "4.0.1",
-    "@skyux/router": "4.2.2",
-    "@skyux/theme": "4.21.1",
-    "codelyzer": "5.2.2",
-    "rxjs": "6.6.7",
+    "@angular-devkit/build-angular": "~12.2.7",
+    "@angular-eslint/builder": "12.4.1",
+    "@angular-eslint/eslint-plugin": "12.4.1",
+    "@angular-eslint/eslint-plugin-template": "12.4.1",
+    "@angular-eslint/schematics": "12.4.1",
+    "@angular-eslint/template-parser": "12.4.1",
+    "@angular/cli": "~12.2.7",
+    "@angular/compiler-cli": "~12.2.7",
+    "@skyux-sdk/documentation-schematics": "^5.0.0-beta.4",
+    "@skyux-sdk/testing": "^5.0.0-beta.2",
+    "@skyux/packages": "^5.0.0-beta.133",
```

Comment:
Range characters should be removed from all `@skyux/*`, `@skyux-sdk/*`, `@angular/*`, and `@angular-devkit/*`.

Also, `@skyux/packages` should be listed in `dependencies`.